root/components/common-macros.tt

```diff
@@ -350,8 +350,12 @@ END -%]
     IF namevar AND action == 'show' AND content != ''
             AND (noescape ? html_unescape(content.remove('</?span\b[^>]*>')) : content) != entity.name;
         options.unshift('name_variation');
-        hover = hover != '' ? l('{name} – {additional_info}', { name => entity.name, additional_info => hover })
-                            : entity.name;
+        IF (entity.sort_name && entity.sort_name == entity.name);
```

Comment:
This is the wrong place to do changes specific for artists, as this is a generic macro that is used for all main entities.